requirements.txt

```diff
@@ -4,102 +4,333 @@
+    #   oslo.i18n
+    #   oslo.serialization
+    #   oslo.utils
+    #   python-cinderclient
+    #   python-keystoneclient
+    #   python-novaclient
+    #   python-openstackclient
+    #   stevedore
+pexpect==4.7.0
+    # via -r requirements.in
+pip-tools==4.3.0
+    # via -r requirements.in
+pluggy==0.13.1
+    # via
+    #   pytest
+    #   tox
+prettytable==0.7.2
+    # via
+    #   -r requirements.in
+    #   cliff
+    #   python-cinderclient
+    #   python-novaclient
+psutil==5.6.6
+    # via -r requirements.in
+ptyprocess==0.6.0
+    # via pexpect
+py==1.9.0
+    # via
+    #   pytest
+    #   tox
+pyasn1==0.2.3
+    # via -r requirements.in
+pycparser==2.20
+    # via cffi
+pyjwt==1.7.1
+    # via -r requirements.in
+pynacl==1.4.0
+    # via paramiko
+pyopenssl==19.0.0
+    # via
+    #   -r requirements.in
+    #   ndg-httpsclient
+pyparsing==2.4.7
+    # via
+    #   cliff
+    #   httplib2
+    #   oslo.utils
+pyperclip==1.8.0
+    # via cmd2
+pytest==3.7.1
+    # via -r requirements.in
+python-cinderclient==7.1.0
+    # via python-openstackclient
+python-dateutil==2.6.0
+    # via
+    #   -r requirements.in
+    #   botocore
+python-keystoneclient==4.1.0
+    # via python-openstackclient
+python-novaclient==17.1.0
+    # via
+    #   -r requirements.in
+    #   python-openstackclient
+python-openstackclient==5.3.1
+    # via -r requirements.in
+pytz==2020.1
+    # via
+    #   oslo.serialization
+    #   oslo.utils
+pyyaml==5.1.2
+    # via
+    #   -r requirements.in
+    #   ansible
+    #   cliff
+    #   openstacksdk
+    #   oslo.config
+requests==2.22.0
+    # via
+    #   -r requirements.in
+    #   apache-libcloud
+    #   keystoneauth1
+    #   oslo.config
+    #   python-cinderclient
+    #   python-keystoneclient
+requestsexceptions==1.4.0
+    # via openstacksdk
+rfc3986==1.4.0
+    # via oslo.config
+s3transfer==0.2.1
+    # via boto3
+sentry-sdk==0.19.5
+    # via -r requirements.in
+simplejson==3.17.0
+    # via
+    #   osc-lib
+    #   python-cinderclient
+    #   python-novaclient
+six==1.14.0
+    # via
+    #   -r requirements.in
+    #   bcrypt
+    #   cliff
+    #   configobj
+    #   cryptography
+    #   debtcollector
+    #   keystoneauth1
+    #   munch
+    #   oslo.i18n
+    #   oslo.utils
+    #   pip-tools
+    #   pynacl
+    #   pyopenssl
+    #   pytest
+    #   python-cinderclient
+    #   python-dateutil
+    #   python-keystoneclient
+    #   python-openstackclient
+    #   tox
+    #   virtualenv
+stevedore==3.1.0
+    # via
+    #   cliff
+    #   keystoneauth1
+    #   osc-lib
+    #   oslo.config
+    #   python-keystoneclient
+    #   python-openstackclient
+toml==0.10.1
+    # via -r requirements.in
+tox==3.0.0
+    # via -r requirements.in
+urllib3==1.25.9
+    # via
+    #   botocore
+    #   requests
+    #   sentry-sdk
+virtualenv==20.0.26
+    # via tox
+wcwidth==0.2.5
+    # via cmd2
+wrapt==1.12.1
+    # via debtcollector
+xmltodict==0.12.0
+    # via -r requirements.in
```

Comment:
@tchaikov this looks new

Reply:
apart from the format change, i don't see anything new here.

Reply:
that's what I'm talking